### DISCUSSION TITLE
englishPCFG.ser.gz (mistakes in output )

### DISCUSSION THREAD

**Discussion by Shasetty** [stanfordnlp/CoreNLP#1360] [Q&A]
I am using, [EnglishPCFG.ser.gz] , model  of (4.5.4 ) version .

by Use the flag -originalDependencies

There  are mistakes in output : -  (possessive(Company-7, ΓÇÖs-8)), 

prep(felt-10, Given-1)
det(circumstances-4, these-2)
amod(circumstances-4, extraordinary-3)
pobj(Given-1, circumstances-4)
det(Company-7, the-6)
poss(board-9, Company-7)
possessive(Company-7, ΓÇÖs-8)
nsubj(felt-10, board-9)
root(ROOT-0, felt-10)
mark(correct-21, that-11)
csubj(correct-21, going-12)
advmod(going-12, ahead-13)
prep(going-12, with-14)
det(issue-16, the-15)
pobj(with-14, issue-16)
aux(correct-21, will-17)
neg(correct-21, not-18)
cop(correct-21, be-19)
advmod(correct-21, morally-20)
ccomp(felt-10, correct-21)

used text is 

Given these extraordinary circumstances, the Company’s
board felt that going ahead with the issue will not be morally correct. 

can you rectify the same and provide a rectified version.

Thanking you in advance

Shankar.P
whatsapp : +91 9845564015 , +91 9343664015
From Bangalore, India

**Comment by AngledLuffa** [comment]
I suspect this is an encoding problem.  You can see what happens when I run the parser on the sentence provided:

```
java edu.stanford.nlp.parser.lexparser.LexicalizedParser -outputFormat typedDependencies edu/stanford/nlp/models/lexparser/englishPCFG.ser.gz foo.txt
[main] INFO edu.stanford.nlp.parser.lexparser.LexicalizedParser - Loading parser from serialized file edu/stanford/nlp/models/lexparser/englishPCFG.ser.gz ... done [0.8 sec].
Parsing file: foo.txt
Parsing [sent. 1 len. 22]: Given these extraordinary circumstances , the Company ’s board felt that going ahead with the issue will not be morally correct .
case(circumstances-4, Given-1)
det(circumstances-4, these-2)
amod(circumstances-4, extraordinary-3)
obl:given(felt-10, circumstances-4)
det(Company-7, the-6)
nmod:poss(board-9, Company-7)
case(Company-7, ’s-8)
nsubj(felt-10, board-9)
root(ROOT-0, felt-10)
mark(correct-21, that-11)
csubj(correct-21, going-12)
advmod(going-12, ahead-13)
case(issue-16, with-14)
det(issue-16, the-15)
obl:with(going-12, issue-16)
aux(correct-21, will-17)
advmod(correct-21, not-18)
cop(correct-21, be-19)
advmod(correct-21, morally-20)
ccomp(felt-10, correct-21)
```

You can try telling the program which encoding to use with

`-encoding utf-8`

**Comment by Shasetty** [comment]
provided solution works.

Thankyou for the response.
